### PR TITLE
added missing translations to pt_BR.json

### PR DESCRIPTION
**Pull request by mvalim**

### CODE REVIEW

**Comment by mvalim** [comment]
Please ignore the comment about removing a duplicated entry, at first I did not noticed the casing difference from the word "Your"

**Comment by luisprmat** [comment]
At the moment it does not seem correct to delete translation strings. It could be lost support for those who use old versions of Laravel or other projects like `laravel/ui`

**Comment by mvalim** [comment]
I agree with you, if you're addressing my first commit, I noticed my mistake and returned the translation string that I removed, and them I proceed to copy the missing translation strings from en.json and translating them.

File: json/pt_BR.json [hunk 1]
```diff
@@ -1,64 +1,169 @@
 {
     "A fresh verification link has been sent to your email address.": "Um novo link de verificação foi enviado para seu endereço de e-mail.",
     "A new verification link has been sent to the email address you provided during registration.": "Um novo link de verificação foi enviado para o endereço de e-mail que você forneceu durante o processo de cadastro.",
+    "Add a new team member to your team, allowing them to collaborate with you.": "Adicionar um novo membro no seu time, permitindo que ele colabore com você.",
+    "Add additional security to your account using two factor authentication.": "Adicione mais segurança à sua conta usando autenticação em dois fatores.",
+    "Add Team Member": "Adicionar membro do time",
```
Comment (jorgercosta) [inline json/pt_BR.json:6]: Should it be "Adicionar membro ao time",

File: json/pt_BR.json [hunk 2]
```diff
@@ -1,64 +1,169 @@
...
+    "API Token Permissions": "Permissões do token da API",
+    "API Token": "Token da API",
+    "API tokens allow third-party services to authenticate with our application on your behalf.": "Tokens da API permitem que serviços de terceiros se autentiquem em nossa aplicação em seu nome.",
+    "API Tokens": "Tokens da API",
+    "Are you sure you want to delete this team? Once a team is deleted, all of its resources and data will be permanently deleted.": "Tem certeza que quer excluir este time? Uma vez excluído, todos os dados e recursos relativos a ele serão excluídos permanentemente.",
+    "Are you sure you want to delete your account? Once your account is deleted, all of its resources and data will be permanently deleted. Please enter your password to confirm you would like to permanently delete your account.": "Tem certeza que quer excluir a sua conta? Uma vez excluída, todos os dados e recursos relativos a ela serão permanentemente excluídos. Por favor informe a sua senha para confirmar que você deseja excluir permanentemente a sua conta.",
+    "Are you sure you would like to delete this API token?": "Tem certeza de que deseja excluir este Token da API?",
```
Comment (jorgercosta) [inline json/pt_BR.json:18]: Uppercase Token vs token

File: json/pt_BR.json [hunk 3]
```diff
@@ -1,64 +1,169 @@
...
+    "Are you sure you would like to leave this team?": "Tem certeza de que deseja sair deste time?",
+    "Are you sure you would like to remove this person from the team?": "Tem certeza de que deseja remover esta pessoa do time?",
     "Before proceeding, please check your email for a verification link.": "Antes de prosseguir, verifique seu e-mail em busca de um link de verificação.",
+    "Browser Sessions": "Sessões do navegador",
     "click here to request another": "clique aqui para solicitar outro",
+    "Close": "Fechar",
+    "Code": "Código",
     "Confirm Password": "Confirmar senha",
+    "Confirm": "Confirmar",
+    "Create a new team to collaborate with others on projects.": "Crie um time para colaborar em outros projetos.",
```
Comment (jorgercosta) [inline json/pt_BR.json:28]: I believe it should be "Crie um time para colaborar com outros  em projetos.",

File: json/pt_BR.json [hunk 4]
```diff
@@ -1,64 +1,169 @@
...
+    "Delete API Token": "Excluir Token da API",
+    "Delete Team": "Excluir Time",
+    "Delete": "Excluir",
+    "Disable": "Desativar",
+    "Done.": "Feito.",
     "E-Mail Address": "Endereço de e-mail",
     "Email Password Reset Link": "Link de redefinição de senha",
     "Email": "Email",
+    "Enable": "Ativar",
+    "Ensure your account is using a long, random password to stay secure.": "Garanta que sua conta esteja usando uma senha longa, e não previsível para se manter seguro.",
```
Comment (jorgercosta) [inline json/pt_BR.json:46]: Consider translating random as "aleatória"

File: json/pt_BR.json [hunk 5]
```diff
@@ -1,64 +1,169 @@
...
+    "For your security, please confirm your password to continue.": "Para sua segurança, por favor confirme a sua senha antes de prosseguir.",
     "Forbidden": "Proibido",
     "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.": "Esqueceu sua senha? Sem problemas. É só nos informar o seu e-mail que nós enviaremos para você um link para redefinição de senha que irá permitir que você escolha uma nova senha.",
     "Forgot Your Password?": "Esqueceu a senha?",
     "Forgot your password?": "Esqueceu a senha?",
     "Go Home": "Ir para o início",
+    "Go to page :page": "Ir para página :page",
     "Hello!": "Olá!",
+    "If necessary, you may logout of all of your other browser sessions across all of your devices. If you feel your account has been compromised, you should also update your password.": "Se necessário, você pode sair de todas suas outras sessões em other browser sessions em todos os seus dispositivos. Se você desconfiar que sua conta foi comprometida, você deve também atualizar a sua senha.",
```
Comment (jorgercosta) [inline json/pt_BR.json:55]: Consider translating "other browser sessions" I suggest "outras sessões de navegador"

File: json/pt_BR.json [hunk 6]
```diff
@@ -1,64 +1,169 @@
...
     "If you did not request a password reset, no further action is required.": "Se você não solicitou essa redefinição de senha, ignore este e-mail.",
-    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Se você estiver com problemas para clicar no botão \":actionText\", copie e cole o URL abaixo\nem seu navegador da web:",
+    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Se você estiver tendo problemas para clicar no botão \":actionText\", copie e cole a URL abaixo\nem seu navegador:",
     "Invalid signature.": "Assinatura inválida.",
+    "Last active": "Última atividade",
+        "Last used": "Usado por último",
+    "Leave Team": "Sair do time",
+    "Leave": "Sair",
     "Login": "Entrar",
+    "Logout Other Browser Sessions": "Desconectar outras sessões de navegadores",
```
Comment (jorgercosta) [inline json/pt_BR.json:66]: Consider "Desconectar outras sessões de navegador"

File: json/pt_BR.json [hunk 7]
```diff
@@ -1,64 +1,169 @@
...
+    "The provided password does not match your current password.": "A senha informada não corresponde à sua senha atual.",
+    "The provided password was incorrect.": "A senha informada está incorreta.",
+    "The provided two factor authentication code was invalid.": "O código de autenticação em dois fatores informado não é válido.",
+    "The team's name and owner information.": "Nome do time e informações do dono.",
     "This action is unauthorized.": "Esta ação não é autorizada.",
+    "This device": "Este dispositivo",
     "This password does not match our records.": "Esta senha não corresponde aos nossos registros.",
     "This password reset link will expire in :count minutes.": "O Link de redefinição de senha irá expirar em :count minutos.",
     "This user already belongs to the team.": "Este usuário já pertence a um time.",
+    "to": "para",
```
Comment (jorgercosta) [inline json/pt_BR.json:143]: Please review the pagination context I believe I should be "até" as in "Mostrando 4 até 10 de 200 resultados"

Here's the original code [(https://github.com/laravel/framework/blob/e63abd8d372f9bad3ca95570cb7e603f3ef28bdc/src/Illuminate/Pagination/resources/views/tailwind.blade.php]((https://github.com/laravel/framework/blob/e63abd8d372f9bad3ca95570cb7e603f3ef28bdc/src/Illuminate/Pagination/resources/views/tailwind.blade.php)

File: json/pt_BR.json [hunk 8]
```diff
@@ -1,64 +1,169 @@
...
     "Please confirm your password before continuing.": "Por favor, confirme sua senha antes de continuar.",
+    "Please copy your new API token. For your security, it won't be shown again.": "Por favor copie seu novo Token da API. Para sua segurança, ele não será mostrado novamente.",
+    "Please enter your password to confirm you would like to logout of your other browser sessions across all of your devices.": "Por favor informe a sua senha para confirmar que você realmente deseja sair de todas sessões de outros navegadores em todos seus dispositivos.",
+    "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Por favor informe o e-mail da pessoa que você deseja adicionar a este time. Esse e-mail deve estar associado a uma conta existente.",
+    "Profile Information": "Informações do perfil",
+    "Profile": "Perfil",
+    "Recovery Code": "Código de recuperação",
     "Regards": "Atenciosamente",
+    "Regenerate Recovery Codes": "Gerar novamente os códigos de verificação",
```
Comment (jorgercosta) [inline json/pt_BR.json:99]: Please consider "Gerar novamente os códigos de recuperação"

File: json/pt_BR.json [hunk 9]
```diff
@@ -1,64 +1,169 @@
...
     "Password": "Senha",
+    "Permanently delete this team.": "Excluir este time permanentemente.",
+    "Permanently delete your account.": "Excluir esta conta permanentemente.",
+    "Permissions": "Permissões",
+    "Photo": "Foto",
     "Please click the button below to verify your email address.": "Por favor, clique no botão abaixo para verificar seu endereço de e-mail.",
     "Please confirm access to your account by entering one of your emergency recovery codes.": "Por favor confirme o acesso à sua conta inserindo um de seus códigos de recuperação de emergência.",
     "Please confirm access to your account by entering the authentication code provided by your authenticator application.": "Por favor confirme o acesso à sua conta informando um código de autenticação fornecido por seu aplicativo autenticador.",
     "Please confirm your password before continuing.": "Por favor, confirme sua senha antes de continuar.",
+    "Please copy your new API token. For your security, it won't be shown again.": "Por favor copie seu novo Token da API. Para sua segurança, ele não será mostrado novamente.",
```
Comment (jorgercosta) [inline json/pt_BR.json:92]: Please consider lowercase "token da API"

File: json/pt_BR.json [hunk 10]
```diff
@@ -1,64 +1,169 @@
...
     "A new verification link has been sent to the email address you provided during registration.": "Um novo link de verificação foi enviado para o endereço de e-mail que você forneceu durante o processo de cadastro.",
+    "Add a new team member to your team, allowing them to collaborate with you.": "Adicionar um novo membro no seu time, permitindo que ele colabore com você.",
+    "Add additional security to your account using two factor authentication.": "Adicione mais segurança à sua conta usando autenticação em dois fatores.",
+    "Add Team Member": "Adicionar membro do time",
+    "Add": "Adicionar",
+    "Added.": "Adicionado.",
+    "All of the people that are part of this team.": "Todas pessoas que fazem parte desse time.",
     "All rights reserved.": "Todos os direitos reservados.",
     "Already registered?": "Já registrado?",
+    "API Token Permissions": "Permissões do token da API",
```
Comment (jorgercosta) [inline json/pt_BR.json:12]: Please consider uppercase Token

File: json/pt_BR.json [hunk 11]
```diff
@@ -29,37 +28,37 @@
...
     "Verify Your Email Address": "Verifique seu endereço de e-mail",
-    "We were unable to find a registered user with this email address.": "We were unable to find a registered user with this email address.",
+    "We were unable to find a registered user with this email address.": "Não pudemos encontrar um usuário com esse endereço de e-mail.",
     "We won't ask for your password again for a few hours.": "Não solicitaremos sua senha novamente por algumas horas.",
     "Whoops!": "Ops!",
     "You are logged in!": "Você já está logado!",
     "You are receiving this email because we received a password reset request for your account.": "Você recebeu esse e-mail porque foi solicitado uma redefinição de senha na sua conta.",
-    "You may not delete your personal team.": "You may not delete your personal team.",
-    "You may not leave a team that you created.": "You may not leave a team that you created.",
+    "You may not delete your personal team.": "Você não pode deletar o seu time pessoal.",
```
Comment (jorgercosta) [inline json/pt_BR.json:167]: Please consider replacing "deletar" with "excluir" to make it more coherent.

File: json/pt_BR.json [hunk 12]
```diff
@@ -13,6 +13,7 @@
     "Forbidden": "Proibido",
     "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.": "Forgot your password? No problem. Just let us know your email address and we will email you a password reset link that will allow you to choose a new one.",
     "Forgot Your Password?": "Esqueceu a senha?",
```
Comment (jorgercosta) [inline json/pt_BR.json:50]: Please consider adding the uppercase "Senha"

File: json/pt_BR.json [hunk 13]
```diff
@@ -1,64 +1,169 @@
...
     "Please confirm your password before continuing.": "Por favor, confirme sua senha antes de continuar.",
+    "Please copy your new API token. For your security, it won't be shown again.": "Por favor copie seu novo Token da API. Para sua segurança, ele não será mostrado novamente.",
+    "Please enter your password to confirm you would like to logout of your other browser sessions across all of your devices.": "Por favor informe a sua senha para confirmar que você realmente deseja sair de todas sessões de outros navegadores em todos seus dispositivos.",
+    "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Por favor informe o e-mail da pessoa que você deseja adicionar a este time. Esse e-mail deve estar associado a uma conta existente.",
+    "Profile Information": "Informações do perfil",
+    "Profile": "Perfil",
+    "Recovery Code": "Código de recuperação",
```
Comment (jorgercosta) [inline json/pt_BR.json:97]: Please consider "Código de Recuperação"

File: json/pt_BR.json [hunk 14]
```diff
@@ -1,64 +1,169 @@
...
     "Please confirm access to your account by entering the authentication code provided by your authenticator application.": "Por favor confirme o acesso à sua conta informando um código de autenticação fornecido por seu aplicativo autenticador.",
     "Please confirm your password before continuing.": "Por favor, confirme sua senha antes de continuar.",
+    "Please copy your new API token. For your security, it won't be shown again.": "Por favor copie seu novo Token da API. Para sua segurança, ele não será mostrado novamente.",
+    "Please enter your password to confirm you would like to logout of your other browser sessions across all of your devices.": "Por favor informe a sua senha para confirmar que você realmente deseja sair de todas sessões de outros navegadores em todos seus dispositivos.",
+    "Please provide the email address of the person you would like to add to this team. The email address must be associated with an existing account.": "Por favor informe o e-mail da pessoa que você deseja adicionar a este time. Esse e-mail deve estar associado a uma conta existente.",
+    "Profile Information": "Informações do perfil",
```
Comment (jorgercosta) [inline json/pt_BR.json:95]: Please consider "Informações do Perfil"

File: json/pt_BR.json [hunk 15]
```diff
@@ -1,64 +1,169 @@
...
+    "Profile": "Perfil",
+    "Recovery Code": "Código de recuperação",
     "Regards": "Atenciosamente",
+    "Regenerate Recovery Codes": "Gerar novamente os códigos de verificação",
     "Register": "Registrar",
     "Remember Me": "Lembre-se de mim",
+    "Remember me": "Lembre-se de mim",
+    "Remove Photo": "Remover Foto",
+    "Remove Team Member": "Remover membro do time",
```
Comment (jorgercosta) [inline json/pt_BR.json:104]: Please consider "Remover Membro do Time"

File: json/pt_BR.json [hunk 16]
```diff
@@ -1,64 +1,169 @@
...
     "Reset Password": "Redefinir senha",
+    "results": "resultados",
+    "Role": "Papel",
+    "Save": "Salvar",
+    "Saved.": "Salvo.",
+    "Select A New Photo": "Selecione uma nova foto",
     "Send Password Reset Link": "Enviar link de redefinição de senha",
     "Server Error": "Erro do servidor",
     "Service Unavailable": "Serviço indisponível",
+    "Show Recovery Codes": "Mostrar códigos de recuperação",
```
Comment (jorgercosta) [inline json/pt_BR.json:117]: Please consider "Mostrar Códigos de Recuperação"

File: json/pt_BR.json [hunk 17]
```diff
@@ -1,64 +1,169 @@
...
     "If you did not create an account, no further action is required.": "Se você não criou uma conta, ignore este e-mail.",
     "If you did not receive the email": "Se você não recebeu o e-mail",
     "If you did not request a password reset, no further action is required.": "Se você não solicitou essa redefinição de senha, ignore este e-mail.",
-    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Se você estiver com problemas para clicar no botão \":actionText\", copie e cole o URL abaixo\nem seu navegador da web:",
+    "If you’re having trouble clicking the \":actionText\" button, copy and paste the URL below\ninto your web browser:": "Se você estiver tendo problemas para clicar no botão \":actionText\", copie e cole a URL abaixo\nem seu navegador:",
     "Invalid signature.": "Assinatura inválida.",
+    "Last active": "Última atividade",
+        "Last used": "Usado por último",
+    "Leave Team": "Sair do time",
```
Comment (jorgercosta) [inline json/pt_BR.json:63]: Please consider "Sair do Time"

File: json/pt_BR.json [hunk 18]
```diff
@@ -1,64 +1,169 @@
...
+    "Save": "Salvar",
+    "Saved.": "Salvo.",
+    "Select A New Photo": "Selecione uma nova foto",
     "Send Password Reset Link": "Enviar link de redefinição de senha",
     "Server Error": "Erro do servidor",
     "Service Unavailable": "Serviço indisponível",
+    "Show Recovery Codes": "Mostrar códigos de recuperação",
+    "Showing": "Mostrando",
+    "Store these recovery codes in a secure password manager. They can be used to recover access to your account if your two factor authentication device is lost.": "Guarde estes códigos de recuperação em um gerenciador de senhas seguro. Eles podem ser usados para recuperar o acesso à sua conta se o dispositivo de autenticação em dois fatores for perdido.",
+    "Switch Teams": "Mudar de time",
```
Comment (jorgercosta) [inline json/pt_BR.json:120]: Please consider "Mudar de Time"

File: json/pt_BR.json [hunk 19]
```diff
@@ -1,64 +1,169 @@
...
+    "Team Details": "Detalhes do time",
```
Comment (jorgercosta) [inline json/pt_BR.json:121]: Please consider "Detalhes do Time"

File: json/pt_BR.json [hunk 20]
```diff
@@ -1,64 +1,169 @@
...
+    "Team Members": "Membros do time",
```
Comment (jorgercosta) [inline json/pt_BR.json:122]: Please consider "Detalhes do Time"

File: json/pt_BR.json [hunk 21]
```diff
@@ -1,64 +1,169 @@
...
+    "Team Name": "Nome do time",
```
Comment (jorgercosta) [inline json/pt_BR.json:123]: Please consider "Nome do Time"

File: json/pt_BR.json [hunk 22]
```diff
@@ -1,64 +1,169 @@
...
+    "Team Owner": "Dono do time",
```
Comment (jorgercosta) [inline json/pt_BR.json:124]: Please consider "Dono do Time"

File: json/pt_BR.json [hunk 23]
```diff
@@ -1,64 +1,169 @@
...
+    "Team Settings": "Configurações do time",
```
Comment (jorgercosta) [inline json/pt_BR.json:125]: Please consider "Configurações do Time"

**Review by jorgercosta — CHANGES_REQUESTED** [review]
Great work,  please reviewer my  comments
Most are corrections that I considered more appropriate, especially related to uppercasing/lowercasing  words 

Please correct the missing translations.

All in all great work!

**Comment by mvalim** [comment]
@jorgercosta I've applied the corrections suggested in my last commit, please confirm if it's ok now. Thanks!

**Review by jorgercosta — APPROVED** [review]
Look's fine, thank's

**Comment by caouecs** [comment]
Obrigado

**Pull request merged**